extensions/adobe/experience/offer-management/offer-representation.schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/offer-representation",
+  "type": "object",
+  "title": "Representation",
+  "description": "Offer Representation",
+  "definitions": {
+    "offer-representation": {
+      "properties": {
+        "xdm:contentType": {
+          "type": "string",
+          "title": "Type",
+          "description": "Content Type"
+        },
+        "xdm:contentValue": {
```

Comment:
What type of value?!

Reply:
See https://github.com/adobe/xdm/pull/245/commits/24e6d701366014e9724462f1834193de7f7d4d7c